conans/client/client_cache.py

```diff
@@ -321,6 +321,15 @@ def package_summary_hash(self, package_ref):
         readed_digest = FileTreeManifest.load(package_folder)
         return readed_digest.summary_hash
 
+    def install_as_editable(self, conan_reference, target_path):
+        linked_folder_sentinel = self._build_path_to_linked_folder_sentinel(conan_reference)
+        save(linked_folder_sentinel, content=target_path)
+
+    def remove_editable(self, conan_reference):
+        if self.installed_as_editable(conan_reference):
```

Comment:
If a check is being done, maybe it should raise if the package is not editable, in case a user inputs a typo or something and doesn't unlink the package.

Reply:
What about returning a True/False to know if the action has been performed and then writing to the command line a warning if it was not in editable mode?

Reply:
It is returning True/False, the warning is closer to the place where the removal is requested.